shell/src/main/java/tachyon/shell/command/LsrCommand.java

```diff
@@ -52,4 +52,9 @@ void runCommand(TachyonURI path) throws IOException {
   public String getUsage() {
     return "lsr <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Displays information for all directories and files under the path specified in args recursively";
```

Comment:
`Displays information for all files and directories under the specified path recursively.`